pkg/types/stack.go

```diff
@@ -0,0 +1,123 @@
+package types
+
+import (
+	"io/ioutil"
+
+	"gopkg.in/yaml.v2"
+)
+
+type Function struct {
+	// The location of the function handler
+	// relative to context
+	Handler string `yaml:"handler"`
+
+	// TODO(Angus) not in the TypeScript
+	Memory int `yaml:"memory,omitempty"`
```

Comment:
We can leave this in 👍  (it should be somewhere in the typescript, but that codebase is a bit of a warren to navigate).